shared/aggregation/attestations/attestations_test.go

```diff
@@ -52,12 +53,8 @@ func TestAggregateAttestations_AggregatePair(t *testing.T) {
 	}
 	for _, tt := range tests {
 		got, err := AggregatePair(tt.a1, tt.a2)
-		if err != nil {
-			t.Fatal(err)
-		}
-		if !ssz.DeepEqual(got, tt.want) {
-			t.Errorf("AggregatePair() = %v, want %v", got, tt.want)
-		}
+		require.NoError(t, err)
+		require.Equal(t, ssz.DeepEqual(got, tt.want), true)
```

Comment:
```suggestion
		require.Equal(t, true, ssz.DeepEqual(got, tt.want))
```
`Equal` signature is `expected, got`, similar to what we generally use when printing out non-equality errors. It is not a big deal per se (check for equality still tests the functionality correctly) -- error message will be a bit confusing though (as it will contain got: want: in wrong order). 
I was aware that this might be confusing, and I had to choose the order, and reviewed how the other libs doing this (e.g. most conventional `testify/assert`), all of the libs I checked used the order `Equal(expected, actual)`, hence my choice.

Reply:
Dismissing my review: https://github.com/prysmaticlabs/prysm/pull/6631